server/src/main/java/com/skillmagnet/Course/Course.java

```diff
@@ -0,0 +1,120 @@
+package com.skillmagnet.Course;
+
+import java.sql.Timestamp;
+
+import org.hibernate.annotations.CreationTimestamp;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "course")
+public class Course {
+    @Id
+    @GeneratedValue
+    private int id;
+
+    private String title;
+    private String description;
+    private String category;
+    private int views;
+    private int thumbsUp;
+    private int thumbsDown;
+
+    @CreationTimestamp
+    private Timestamp createdAt;
+    @CreationTimestamp
+    private Timestamp status;
```

Comment:
The `@CreationTimestamp` is fine with me - there's tons of crazy annotations that do fun stuff. Nice find lol.

:question: Is there any reason you did `Timestamp` vs a `DateTime`? I looked it up and I'm not oppposed to the Timestamp as I don't fully understand the differences. Just I've always seen a DateTime used lol. If you can still format it into a date (e.g. Jan 10th, 2024 8:05:00) then I'm good with it.

Reply:
Switched to `LocalDateTime` in the newest commit.

I couldn't find anything called just DateTIme but referenced this article [here](https://www.baeldung.com/java-8-date-time-intro)

LocalDatetime seems nice, lets you add/subtract minutes, hours, days, etc, which could come in handy. Section 7 of that article shows using the `DateTimeFormatter` to get custom formats.